apps/public-docsite/src/components/Site/Site.tsx

```diff
@@ -460,21 +463,9 @@ export class Site<TPlatforms extends string = string> extends React.Component<
     const { siteDefinition } = this.props;
     const { platforms } = siteDefinition;
 
-    const newPagePath = removeAnchorLink(location.hash);
+    const newPagePath = getNormalizedPath();
     // Top level path (Controls, Get started or Styles)
-    let newPageTopLevel = '';
```

Comment:
This was removed because it mostly duplicates functionality of the `getSiteArea` helper